homeassistant/helpers/condition.py

```diff
@@ -925,19 +925,6 @@ def zone(
     ):
         return False
 
-    latitude = entity.attributes.get(ATTR_LATITUDE)
-    longitude = entity.attributes.get(ATTR_LONGITUDE)
-
-    if latitude is None:
-        raise ConditionErrorMessage(
-            "zone", f"entity {entity_id} has no 'latitude' attribute"
-        )
-
-    if longitude is None:
-        raise ConditionErrorMessage(
-            "zone", f"entity {entity_id} has no 'longitude' attribute"
-        )
-
```

Comment:
This is not correct. `zone_cmp.in_zone` requires a latitude and longitude to determine if an entity is in a zone.